WordPressKit/ReaderPostServiceRemote+Cards.swift

```diff
@@ -7,14 +7,16 @@ extension ReaderPostServiceRemote {
     ///
     /// - Parameter topics: an array of String representing the topics
     /// - Parameter page: a String that represents a page handle
+    /// - Parameter sortingOption: a String that represents a sorting option
     /// - Parameter success: Called when the request succeeds and the data returned is valid
     /// - Parameter failure: Called if the request fails for any reason, or the response data is invalid
     public func fetchCards(for topics: [String],
                            page: String? = nil,
+                           sortingOption: String? = nil,
```

Comment:
Maybe we should use an `enum` (with a `String rawValue`) instead of just passing a string to make it less error prone? One of the cases could also be a `noSorting` option that should then be the default.

Reply:
Sure! I made it to be `String?` so that in the future we can pass new values without the need to update `WordPressKit`. But I can use the enum here 👍